### PR TITLE
Revamp UI in moderator and student dashboard

### LINKED ISSUE #616: Moderator Dashboard needs to be enhanced/refactored
- Add pagination
- Remove inline scrolling
- Add more info

### LINKED ISSUE #617: Main student Dashboard
- [ ] Make course title proper.
- [ ] Add pagination for the default courses and enrolled courses.

### LINKED ISSUE #618: Fix UI/UX inside course page in student dashboard
- [ ] Make the sidebar proper and ability to hide.
- [ ] Show course and module information properly.

### PR DESCRIPTION
**Pull request by adityacp**
This PR closes #616, closes #617, closes #618

### CODE REVIEW

**Comment by pyfossee** [comment]
> * Regrade Takes a lot of time to Load
This will be changed in a separate PR.
> * Increase the paginator width to show more page numbers than just 3. User can select from many options than just 3
Will try to change this and add more page numbers
> * exam/reset/changepassword/ does not have the nav logout dropdown
Need to check this because the change password template is common for both moderator and students. 
> * Paginator in Students interface if required.
I feel this is ok for now if it is required then maybe we can add later.
> * Small fonts in the course_module page
Can you show me this? In my system, it looks ok.
> * Inconsistent use of font size in different pages of the web app
I did not change the font size on every page it is common for all.

File: yaksh/templates/yaksh/moderator_dashboard.html [hunk 1]
```diff
@@ -1,116 +1,121 @@
...
+                                {% endif %}
+                            </div>
+                            <div class="col-md-3">
+                                <a href="{% url 'yaksh:course_detail' course.id %}" class="btn btn-primary">
+                                    <i class="fa fa-tasks"></i>
+                                    Manage Course
+                                </a>
+                            </div>
+                            <div class="col-md">
+                                <a class="card-link btn btn-info" data-toggle="collapse" href="#collapse{{course.id}}">
```
[the remark this answers is not in the document]
Comment (adityacp) [reply yaksh/templates/yaksh/moderator_dashboard.html]: I did not want to add any custom styling. Button color is bootstrap default.

**Comment by adityacp** [comment]
> * Regrade Takes a lot of time to Load
This will be changed in a separate PR
> * Increase the paginator width to show more page numbers than just 3. User can select from many options than just 3
I will try to fix this and increase the page numbers and reduce the pages.
> * exam/reset/changepassword/ does not have the nav logout dropdown
I need to check this as the template is common for both moderators and students.
> * Paginator in Students interface if required.
I feel it is ok for now to have, it can be added later
> * Small fonts in the course_module page
> * Inconsistent use of font size in different pages of the web app
I did not see any inconsistent font size in my system. Can you just show me?

File: yaksh/views.py [hunk 2]
```diff
@@ -408,30 +406,21 @@ def prof_manage(request, msg=None):
...
-                else:
-                    answerpaper.delete()
-
-    context = {'user': user, 'courses': courses,
-               'trial_paper': trial_paper, 'msg': msg
-               }
+    courses = Course.objects.get_queryset().filter(
+        Q(creator=user) | Q(teachers=user),
+        is_trial=False).distinct().order_by("-active")
+    paginator = Paginator(courses, 30)
```
[the remark this answers is not in the document]
Comment (adityacp) [reply yaksh/views.py]: I will check this.

**Comment by CruiseDevice** [comment]
LGTM

**Pull request merged**